scripts/container/utils

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+wait_for_es_to_start() {
+  count=1
+  max_count=5
+  wait_seconds=30
+  ES_HOST=${ES_HOST:-es}
+  ES_PORT=${ES_PORT:-9200}
+
+  echo "Waiting for a response from http://$ES_HOST:$ES_PORT"
+  while ! wget -q -O /dev/null http://"$ES_HOST":"$ES_PORT" ; do
```

Comment:
The mechanism used to check if ES is ready in the nearby-services-api checks the health status of ES rather than seeing if the main page responds, as it's possible for ES to be up and running without the indexes having finished building.

If you are seeing sporadic failures in rancher, this might be the culprit.